code/datums/wires/explosive.dm

```diff
@@ -26,6 +26,18 @@
 	if(G.stage == GRENADE_WIRED)
 		return TRUE
 
+/datum/wires/explosive/chem_grenade/on_pulse(index)
+	var/obj/item/grenade/chem_grenade/G = holder
+	if(G.stage != GRENADE_READY)
+		return
+	. = ..()
+
+/datum/wires/explosive/chem_grenade/on_cut(index, mend)
+	var/obj/item/grenade/chem_grenade/G = holder
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names